### PR TITLE
Classes and methods of the modules

### PR DESCRIPTION
**Pull request by Rodrigo-Flores**
Overwritting classes and methods of OctaSat modules.

### CODE REVIEW

**Comment by Rodrigo-Flores** [comment]
Version ready to test and debug. Main version2 has been created to implement module classes and methods.

File: main-v2.py [hunk 1]
```diff
@@ -0,0 +1,35 @@
+from time import sleep
+from .modules.mainModule import NEO, HDC, BMP, Buzzer
```
Comment (CxrlosKenobi) [inline main-v2.py:2]: Not an absolute import :eyes:

File: modules/NEO6M.py [hunk 2]
```diff
@@ -1,46 +1,88 @@
...
+        dat = ser.readline().decode()
+        mylat,mylon = self.paser(dat)
 
-    dat = ser.readline().decode()
-    mylat,mylon = parseGPS(dat)
+        return mylat, mylon
 
-    return mylat, mylon
+# NEO = NEO()
+# NEO.GPS()
```
Comment (CxrlosKenobi) [inline modules/NEO6M.py:88]: Files must have a newline at EOF

**Review by CxrlosKenobi — COMMENTED** [review]
Let's wait until we can test the code before merging it into the `main` branch, everything looks good though.

File: main-v2.py (hunk 1 above)
Comment (Rodrigo-Flores) [reply main-v2.py:2]: You try no say that the import should be something like ".OctaSat_Demo.modules.mainModule"?

File: modules/NEO6M.py (hunk 2 above)
Comment (Rodrigo-Flores) [reply modules/NEO6M.py:88]: I didn't catch it. You mean to put an example line in all of the files? Anyway, this code isn't the "final version". You can find it in `.modules.mainModule`.

File: main-v2.py (hunk 1 above)
Comment (CxrlosKenobi) [reply main-v2.py]: It would be `modules.mainModule import NEO` ...

**Pull request closed without merging**